scripts/build.py

```diff
@@ -609,6 +611,80 @@ def build_files(source, target, file_hashes):
             print e
 
 
+def build_HTML_files(source, target, file_hashes):
```

Comment:
Can we rearchitect to reduce the overlap between this and build_files() above?

Reply:
Done. PTAL!